src/upgrade.py

```diff
@@ -199,6 +200,7 @@ def _on_upgrade_granted(self, event: UpgradeGrantedEvent) -> None:  # noqa: C901
             # stop cron daemon to be able to query `error.log`
             set_cron_daemon("stop")
             self.charm._mysql.start_mysqld()
+            self.charm._mysql.install_plugins()
             self.charm._mysql.setup_logrotate_and_cron()
```

Comment:
It's a per instance affair, no impact.
On rollback with an older snap without the plugin, mysqld will fail to load the `.so` file (not found), generating some logs in `error.log` file.
It will gracefully start though.